app/main/views/g_cloud.py

```diff
@@ -292,7 +293,7 @@ def saved_search_overview(framework_framework):
     closed_projects = []
 
     for project in projects:
-        if project['active']:
+        if project['lockedAt'] is None:
```

Comment:
Let's put this back - the distinction for this project list page is between open procurements and closed ones. Saved searches are still 'open' in this sense - they have neither been awarded nor archived, so they are 'active'.

if we want to show the locked status here, then that's a different thing?